<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="TunnusKentta">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SalasanaKentta">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MongoDB Tunnus</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MongoDB Salasana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DBNappi">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Testaa Yhteys</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="yhteysViestiKentta">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="kansioNappi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valitse ladattava kansio</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="kansioKentta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="analysoiNappi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Analysoi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lataaNappi">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Tallenna kohteisiin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="piirraNappi">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>N채yt채 K채ppyr채</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pilviCheck">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pilveen?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="csvCheck">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="projektiKentta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Projekti/testi tunnus</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tallenussijainti CSV:lle</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>20</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
